src/pages/BuyGMX/BuyGMX.js

```diff
@@ -1,29 +1,210 @@
+    networks: [AVALANCHE],
+    link: "https://bridge.avax.network/",
+  },
+  {
+    name: "Hop",
+    icon: "ic_hop.svg",
+    networks: [ARBITRUM],
+    link: "https://app.hop.exchange/send?token=ETH&sourceNetwork=ethereum&destNetwork=arbitrum",
+  },
+  {
+    name: "Bungee",
+    icon: "ic_bungee.png",
+    networks: [ARBITRUM, AVALANCHE],
+    link: "https://multitx.bungee.exchange",
+  },
+  {
+    name: "Multiswap",
+    icon: "ic_multiswap.svg",
+    networks: [ARBITRUM, AVALANCHE],
+    link: "https://app.multichain.org/#/router",
+  },
+  {
+    name: "O3",
+    icon: "ic_o3.png",
+    networks: [ARBITRUM, AVALANCHE],
+    link: "https://o3swap.com/",
+  },
+  {
+    name: "Across",
+    icon: "ic_across.svg",
+    networks: [ARBITRUM, AVALANCHE],
+    link: "https://across.to/",
+  },
+  {
+    name: "FTX",
+    icon: "ic_ftx.svg",
+    networks: [ARBITRUM],
+    link: "https://ftx.com/trade/ETH/USD",
+  },
+];
+
+const CENTRALISED_EXCHANGES = [
+  {
+    name: "Binance",
+    icon: "ic_binance.svg",
+    link: "https://www.binance.com/en/trade/GMX_USDT?_from=markets",
+  },
+  {
+    name: "Bybit",
+    icon: "ic_bybit.svg",
+    link: "https://www.bybit.com/en-US/trade/spot/GMX/USDT",
+  },
+  {
+    name: "FTX",
+    icon: "ic_ftx.svg",
+    link: "https://ftx.com/trade/GMX/USD",
+  },
+  {
+    name: "Crypto.com",
+    icon: "crypto_com.svg",
+    link: "https://crypto.com/exchange/trade/spot/GMX_USDT",
+  },
+  {
+    name: "Kucoin",
+    icon: "ic_kucoin.svg",
+    link: "https://www.kucoin.com/trade/GMX-USDT",
+  },
+  {
+    name: "Huobi",
+    icon: "ic_huobi.svg",
+    link: "https://www.huobi.com/en-us/exchange/gmx_usdt/",
+  },
+];
+
+const DECENTRALISED_AGGRIGATORS = [
+  {
+    name: "1inch",
+    icon: "ic_1inch.svg",
+    link: "https://app.1inch.io/",
+    networks: [ARBITRUM, AVALANCHE],
+  },
+  {
+    name: "Matcha",
+    icon: "ic_matcha.png",
+    link: "https://www.matcha.xyz/",
+    networks: [ARBITRUM, AVALANCHE],
+  },
+  {
+    name: "Paraswap",
+    icon: "ic_paraswap.svg",
+    link: "https://www.paraswap.io/",
+    networks: [ARBITRUM, AVALANCHE],
+  },
+  {
+    name: "Firebird",
+    icon: "ic_firebird.png",
+    link: "https://firebird.finance/",
+    networks: [ARBITRUM, AVALANCHE],
+  },
+  {
+    name: "OpenOcean",
+    icon: "ic_openocean.svg",
+    link: "https://openocean.finance/",
+    networks: [ARBITRUM, AVALANCHE],
+  },
+  {
+    name: "DODO",
+    icon: "ic_dodo.svg",
+    link: "https://dodoex.io/",
+    networks: [ARBITRUM, AVALANCHE],
+  },
+  {
+    name: "Odos",
+    icon: "ic_odos.png",
+    link: "https://app.odos.xyz/",
+    networks: [ARBITRUM],
+  },
+  {
+    name: "Slingshot",
+    icon: "ic_slingshot.svg",
+    link: "https://slingshot.finance/",
+    networks: [ARBITRUM],
+  },
+  {
+    name: "Yieldyak",
+    icon: "ic_yield_yak.png",
+    link: "https://yieldyak.com/swap",
+    networks: [AVALANCHE],
+  },
+];
+
 export default function BuyGMX() {
```

Comment:
It would be nice to translate the component into Typescript

Reply:
Done!